server/bed.js

```diff
@@ -189,15 +189,20 @@ bed.listBed_ = function(bedPath) {
   var ret = [];
   var files = fs.readdirSync(folder);
   files.forEach(function(file) {
-    if (file.indexOf('.txt') != -1) {
-      var fname = file.substr(0, file.length - 4);
-      var content = fs.readFileSync(folder + file, 'utf8')
-        .toString().split('\n');
-      var bedName = content[0];
-      var description = content.slice(1).join('');
+    if (file.indexOf('.data') != -1) {
```

Comment:
It depends on the data. Generally I don't think the original name could contain '.data' in the middle, but for safety I will change it to check the ending of the string.